.eslintrc.json

```diff
@@ -16,6 +16,8 @@
   "plugins": ["@typescript-eslint", "jsx-a11y", "prettier", "simple-import-sort"],
   "ignorePatterns": ["lib"],
   "rules": {
+    "@typescript-eslint/ban-ts-comment": "off",
+    "@typescript-eslint/no-non-null-assertion": "off",
```

Comment:
Disabling these rules, especially `no-non-null-assertion` since there can be situations where we want to do `variable!`